include/fmt/format.h

```diff
@@ -3395,6 +3395,43 @@ inline typename buffer_context<Char>::type::iterator format_to(
                               basic_format_args<context>(as));
 }
 
+namespace internal {
+
+template<typename... Ts>
+struct void_ { typedef void type; };
+
+template <typename T, typename Enable = void_<>::type>
```

Comment:
Why `void_<>::type` and not just `void`? A comment might be good if this is required.

Reply:
Well spotted, this is no longer necessary !